README.md

```diff
@@ -0,0 +1,158 @@
+# device-coap-c
+
+EdgeX device service for CoAP-based REST protocol
+
+This device service allows a 3rd party sensor application to push data into EdgeX via CoAP. Like HTTP, CoAP provides REST based access to resources, but CoAP is more compact for use in constrained IoT devices.
+
+The device-coap-c service (_device-coap_ for short) is modeled after the HTTP based [device-rest-go](https://github.com/edgexfoundry/device-rest-go) service, and runs over UDP. The current implementation is meant for one-way communication from a device into EdgeX by posting readings asynchronously. Synchronous interaction initiated by EdgeX can be added in the future.
+
+device-coap uses DTLS for secure communication to devices. It is written in C, and relies on the well known [libcoap](https://libcoap.net/) library.
+
```

Comment:
Add link to meeting and slides from EdgeX Device Working Group on 2020-10-12?

Reply:
Sure. They can be found on https://wiki.edgexfoundry.org/display/FA/Device+and+Device+SDK+Working+Group